cmd/winch/commands/ci.go

```diff
@@ -134,6 +134,17 @@ func ci(ctx context.Context) error {
 
 	cmd := config.CommandFromContext(ctx)
 
+	dryRun, err := cmd.Flags().GetBool("dry-run")
+	if err != nil {
+		return err
+	}
+
+	if dryRun {
+		fmt.Println("Performing a dry run build")
+	} else if os.Getenv("CI") != "true" {
+		return fmt.Errorf("must be running in a CI environment")
```

Comment:
what is reasoning here?

Reply:
without dryrun it does potentially destructive things like tagging and releasing artifacts.